src/libse/Http/LegacyHttpClient.cs

```diff
@@ -112,37 +112,7 @@ public void Dispose()
             _httpClient?.Dispose();
         }
 
-        public static void CopyTo(Stream source, Stream destination, int bufferSize)
```

Comment:
This method is removed as the framework as a similar one!
ps: The one in framework also perform validations